functions/Invoke-DbaAdvancedRestore.ps1

```diff
@@ -333,6 +335,8 @@ function Invoke-DbaAdvancedRestore {
                                 CompressedBackupSize   = if ([bool]($backup.psobject.Properties.Name -contains 'CompressedBackupSize')) { ($backup | Measure-Object -Property CompressedBackupSize -Sum).Sum } else { $null }
                                 Script                 = $script
                                 BackupFileRaw          = ($backups.Fullname)
+                                FileRestoreTime        = New-TimeSpan -Seconds ((Get-Date)-$FileRestoreStartTime).TotalSeconds
```

Comment:
Shouldn't this be `DbaTimeSpan` and utilize something like total milliseconds to get more precision in timing?

Reply:
Very true. Actually, I think just [DbaTimeSpan] without milliseconds. i may merge and fix

Reply:
mmm, no after some testing, this looks best 👍